### PR TITLE
Translate cookbook docs to Chinese

### PR DESCRIPTION
**Pull request by dingjs**
Translate cookbook docs to Chinese. I already transfered the minio project's docs,next step would be translating the sdk docs.The Chinese developers would have the official Chinese docs finally.

### CODE REVIEW

**Comment by deekoder** [comment]
As per our conversation @dingjs  will get it reviewed locally for chinese content. Otherwise this pr looks good to me.

File: docs/zh_CN/running-deis-workflow-with-minio.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+[Deis Workflow](https://deis.com/)是一个开源的[PaaS](https://en.wikipedia.org/wiki/Platform_as_a_service) ，可以很容易地在自己的服务器上部署和管理应用程序。Workflow建立于[Kubernetes](http://kubernetes.io/)和[Docker](https://www.docker.com/)基础上，提供一个轻量级的PaaS，受[Heroku](https://www.heroku.com/)启发的工作流。Workflow有多个模块化比较好的组件（请看 <https://github.com/deis>），它们之间使用Kubernetes system和一个对象存储服务进行通信。它有良好的可配置性，可以配置为使用多种云存储服务，像[Amazon S3](https://aws.amazon.com/s3/), [Google Cloud Storage](https://cloud.google.com/storage/) ， [Microsoft Azure Storage](https://azure.microsoft.com/en-us/services/storage/)，当然，还有Minio。我们目前不会建议你在Deis Workflow生产环境上使用Minio，目前Minio可以做为快速安装一个Deis Workflow集群，用于演示、开发、测试的方案。事实上，我们默认提供了装有Minio的Deis Workflow。
+
+要使用它，请按照<https://docs-v2.readthedocs.io/en/latest/installing-workflow/>中的说明进行操作。完成安装后，请按以下三种方法进行部署：
+
+- [Buildpack部署](https://docs-v2.readthedocs.io/en/latest/applications/using-buildpacks/)
+- [Dockerfile部署](https://docs-v2.readthedocs.io/en/latest/applications/using-dockerfiles/)
+- [Docker Image部署](https://docs-v2.readthedocs.io/en/latest/applications/using-docker-images/)
+
+所有这三种部署方法，以及Workflow内部广泛使用Minio：
```
Comment (yihuaf) [inline docs/zh_CN/running-deis-workflow-with-minio.md]: 使用 -> 使用了

File: docs/zh_CN/running-deis-workflow-with-minio.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+- Buildpack部署使用Minio存储代码和[slugs](https://devcenter.heroku.com/articles/slug-compiler)
```
Comment (yihuaf) [inline docs/zh_CN/running-deis-workflow-with-minio.md]: 使用了

File: docs/zh_CN/running-deis-workflow-with-minio.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+- Dockerfile部署使用Minio存储Dockerfiles和关联的artifacts
```
Comment (yihuaf) [inline docs/zh_CN/running-deis-workflow-with-minio.md]: 使用了

File: docs/zh_CN/running-deis-workflow-with-minio.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+- Docker Image部署使用Minio作为运行Workflow的内部Docker registry的后备存储
```
Comment (yihuaf) [inline docs/zh_CN/running-deis-workflow-with-minio.md]: 使用了

File: docs/zh_CN/s3cmd-with-minio.md [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+```
+
+### 拷贝一个文件到存储桶
+
+```sh
+s3cmd put newfile s3://testbucket
+upload: 'newfile' -> 's3://testbucket/newfile'  
+```
+
+### 拷贝一个文件到本地文系统
```
Comment (yihuaf) [inline docs/zh_CN/s3cmd-with-minio.md]: 文件系统

File: docs/zh_CN/s3cmd-with-minio.md [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+
+```sh
+s3cmd get s3://testbucket/newfile
+download: 's3://testbucket/newfile' -> './newfile'
+```
+
+### 同步同地文件/文件夹到存储桶
```
Comment (yihuaf) [inline docs/zh_CN/s3cmd-with-minio.md]: 本地

File: docs/zh_CN/s3fs-fuse-with-minio.md [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+运行`s3fs`挂载存储桶，使用之前命令中的Minio认证信息。
+
+```
+s3fs <bucket> /s3 -o passwd_file=/etc/s3cred,use_path_request_style,url=http://minio-server:9000
+
+```
+
+s3fs与Minio一起使用时需要`use_path_request_style`。如果您不使用它，则无法在挂载的目录中查看或复制文件。
+
+检查桶是否使用mount命令挂载成功：
```
Comment (yihuaf) [inline docs/zh_CN/s3fs-fuse-with-minio.md]: 存储桶

File: docs/zh_CN/setup-nginx-proxy-with-minio.md [hunk 8]
```diff
@@ -0,0 +1,90 @@
...
+ location ~^/files {
+   proxy_buffering off;
+   proxy_set_header Host $http_host;
+   proxy_pass http://localhost:9000;
+ }
+```
+
+注意:
+
+* 用你自己的服务名替换`http：// localhost：9000`。
```
Comment (yihuaf) [inline docs/zh_CN/setup-nginx-proxy-with-minio.md]: url的间距看上去有问题

File: docs/zh_CN/minio-as-eucalyptus-object-storage.md [hunk 9]
```diff
@@ -0,0 +1,118 @@
...
+部署基本的Minio服务非常简单。在本示例中，我使用了运行Ubuntu 16.04.1 LTS（Xenial Xerus）的Eucalyptus实例（虚拟机）。
+
+
+```
+$ wget https://dl.minio.io/server/minio/release/linux-amd64/minio
+$ chmod +x minio
+$ ./minio server ~/MinioBackend
+```
+
+我们应该看到如下的贼拉有用的输出信息，
```
[the remark this answers is not in the document]
Comment (dingjs) [reply docs/zh_CN/minio-as-eucalyptus-object-storage.md]: 非常感谢如此细心的review,我将在明天修正review的问题。
PS：并不是东北人，只是觉得API文档比较枯燥，所以想让研发人员看的时候有个彩蛋。

File: README_zh_CN.md [hunk 10]
```diff
@@ -0,0 +1,49 @@
...
+- [将PostgreSQL备份存储到Minio Server](./docs/zh_CN/store-postgresql-backups-in-minio.md)
+- [restic结合Minio Server](./docs/zh_CN/restic-with-minio.md)
+- [使用S3cmd操作Minio Server](./docs/zh_CN/s3cmd-with-minio.md)
+- [使用fluentd插件聚合Apache日志](./docs/zh_CN/aggregate-apache-logs-with-fluentd-and-minio.md)
+- [使用Certbot生成Let's Encrypt证书](./docs/zh_CN/generate-lets-encypt-certificate-using-certbot-for-minio.md)
+- [给Minio Server设置Caddy proxy](./docs/zh_CN/setup-caddy-proxy-with-minio.md)
+- [给Minio Server设置Nginx代理](./docs/zh_CN/setup-nginx-proxy-with-minio.md)
+- [给Minio Server设置Apache HTTP proxy](./docs/zh_CN/setup-apache-http-proxy-with-minio.md)
+- [如何使用Træfɪk运行多个Minio服务](./docs/zh_CN/multiple-minio-servers-with-traefik.md)
+- [如何使用Træfɪk loadbalancer和Docker swarm运行分布式Minio](./docs/zh_CN/distributed-minio-with-traefik-as-loadbalancer-in-swarm.md)
```
[the remark this answers is not in the document]
Comment (dingjs) [reply README_zh_CN.md]: yes,it should be lower case.

File: docs/zh_CN/alluxio-with-minio.md [hunk 11]
```diff
@@ -0,0 +1,61 @@
...
+cd alluxio-<VERSION>-<DISTRIBUTION>
+```
+
+拷贝下面的示例模板，创建Alluxio的配置文件。
+
+```sh
+cp conf/alluxio-site.properties.template conf/alluxio-site.properties
+```
+
+为你的部署适当地修改Alluxio配置文件。下面示例是在本地运行Alluxio和Minio的示例配置。
```
[the remark this answers is not in the document]
Comment (dingjs) [reply docs/zh_CN/alluxio-with-minio.md]: 已修改，并去掉将会，国外喜欢说You will need XXX。国内一般不说你将会XXX。

File: docs/zh_CN/presigned-get-download-from-browser.md [hunk 12]
```diff
@@ -0,0 +1,61 @@
...
+    secretKey: 'zuf+tfteSlswRu7BJ86wekitnifILbZam1KYY3TG'
+})
+```
+
+为了发这个请求，我们需要创建一个Minio client,并且输入咱们的认证信息。通过这个clinet,你可以上传下载文件，如果觉得这还不够，请看[API](https://github.com/minio/minio-js/blob/master/docs/API.md)。
+
+示例中给出的是Minio测试服务的真实认证信息，玩玩呗，来都来了!
+
+```js
+// express is a small HTTP server wrapper, but this works with any HTTP server
```
[the remark this answers is not in the document]
Comment (dingjs) [reply docs/zh_CN/presigned-get-download-from-browser.md]: 代码块中的英文注释都需要翻译成中文注释？

File: docs/zh_CN/alluxio-with-minio.md (hunk 11 above)
[the remark this answers is not in the document]
Comment (yihuaf) [reply docs/zh_CN/alluxio-with-minio.md]: 没问题

File: docs/zh_CN/minio-as-eucalyptus-object-storage.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (yihuaf) [reply docs/zh_CN/minio-as-eucalyptus-object-storage.md]: Yes I had a good laugh when I saw those. Overall it was an admirable effort. Looking forward to the changes.

File: docs/zh_CN/presigned-get-download-from-browser.md (hunk 12 above)
[the remark this answers is not in the document]
Comment (yihuaf) [reply docs/zh_CN/presigned-get-download-from-browser.md]: It was more for consistency. You translated almost all the comments, and this is one of the few you didn't. Maybe for consistency, just translate these as well?

File: docs/zh_CN/minio-as-eucalyptus-object-storage.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (harshavardhana) [reply docs/zh_CN/minio-as-eucalyptus-object-storage.md]: Non chinese speaker trying to understand the joke here :smile:

**Pull request merged**